### PR TITLE
Onboard create-data-object task

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

This task will replace create-datavolume-from-manifest.
This time it is written in golang with its own task image.

**Release note**:

```release-note
Onboard create-data-object task
```

### CODE REVIEW

File: modules/create-datavolume-from-manifest/build/create-datavolume-from-manifest/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal AS builder
+RUN microdnf install -y golang-1.16.* && microdnf clean all
```
Comment (ksimon1) [inline modules/create-datavolume-from-manifest/build/create-datavolume-from-manifest/Dockerfile:2]: can you please bump golang to 1.18?

Comment (0xFelix) [reply modules/create-datavolume-from-manifest/build/create-datavolume-from-manifest/Dockerfile:2]: This is only for onboarding purposes. The golang version will be bumped in the PR with the actual changes.

**Pull request merged**